synapse/lib/cell.py

```diff
@@ -467,25 +479,34 @@ class Cell(s_nexus.Pusher, s_telepath.Aware):
             'description': 'Record all changes to the cell.  Required for mirroring (on both sides).',
             'type': 'boolean'
         },
+        'dmon:listen': {
+            'description': 'A config-driven way to specify the telepath bind URL.',
+            'type': ['string', 'null'],
+        },
+        'https:port': {
+            'description': 'A config-driven way to specify the HTTPS port.',
+            'type': ['integer', 'null'],
+        },
     }
 
-    async def __anit__(self, dirn, conf=None, readonly=False, *args, **kwargs):
+    async def __anit__(self, dirn, conf=None, readonly=False):
 
+        # phase 1
```

Comment:
Is there a reason we did not just call these functions
```
async def phase1(self, ....):...
async def phase2(self, ....):...
```

then in `__anit__(self,....)`

we end up calling 
```
await self.phase1()
await self.phase2()
```

seems like it would be a very clearly defined startup, as opposed to relying on inline comments.

Reply:
Having a service-overridden method named "phase3" not only locks us into that ordering sequence (bad) but also has no "self-describing" nature like `initServiceStorage` does.  I've been burned by needing to re-order startup methods with hard-coded names that imply an order before.